optimum/intel/openvino/quantization.py

```diff
@@ -334,7 +335,7 @@ def _quantize_ovcausallm(
                 quantization_config = OVWeightQuantizationConfig(mode=nncf.CompressWeightsMode.INT8_SYM)
                 self.model.model = nncf.compress_weights(self.model.model)
             else:
-                compress_decoder_weights(self.model, quantization_config)
+                _int4_weight_only_quantization(self.model, quantization_config)
```

Comment:
We also need to support INT8_SYM and INT8_ASYM the same way as you did it for INT4 precision. I mean something like `mode=CompressWeightsMode.INT8_SYM if config.sym else CompressWeightsMode.INT8_ASYM`

Reply:
yes totally makes sense !

Reply:
added `_weight_only_quantization` to support both